corehq/apps/performance_sms/templates/performance_sms/list_performance_configs.html

```diff
@@ -34,7 +34,7 @@
                 <td>{{ performance_config.schedule.interval }}</td>
                 <td>{{ performance_config.template }}</td>
                 <td class="btn-group">
-                    <a class="btn btn-success" href="{% url "performance_sms.edit_performance_config" domain performance_config.get_id %}">
+                    <a class="btn btn-success" href="{% if performance_config.is_advanced %}{% url "performance_sms.edit_performance_config_advanced" domain performance_config.get_id %}{% else %}{% url "performance_sms.edit_performance_config" domain performance_config.get_id %}{% endif %}">
```

Comment:
If there will be other places referencing this view, I could see this approach being error prone - you must always remember to check `is_advanced`.  Maybe they could both use a view which thinly dispatches to the approriate actual view.